tools/civic/civic_qrt.xml

```diff
@@ -0,0 +1,82 @@
+<tool name="CIViC Query Report Tool" id="civic_qrt" version="0.1">
+  <description>CIViC Query Report Tool reads a MAF file, calls CIViC for each line therein, and outputs a table in HTML format for easy inclusion into downstream reports</description>
+  <command interpreter="python">
+   civic_qrt.py
+   -i "${input_file}"
+   -o "${out_file}"
+   -c "${host}"
+   -col "${entrez_id_column}"
+  </command>
+  <inputs>
+    <param name="input_file" type="data" format="maf" 
+      label="MAF File from Oncotator" help="If empty, upload or import a MAF dataset."/>
```

Comment:
It will run in the remote site where the input dataset is. From that remote site we can potentially query every other site that runs CIViC server. The out_file alone should be brought back to Central Site using the fetch tool so we can click on the links for the respective genes/variants that were queried in any of the remote site.